frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -721,19 +725,19 @@ const MonitoringDashboardsPage: React.FC<MonitoringDashboardsPageProps> = ({ mat
         </Helmet>
       )}
       <div className="co-m-nav-title co-m-nav-title--detail">
-        {!namespace && <HeaderTop />}
-        <div className="monitoring-dashboards__variables">
-          <div
-            className={cx('monitoring-dashboards__dropdowns', {
-              'monitoring-dashboards__variable-dropdowns': namespace,
-            })}
-          >
+        {!namespace && <HeaderTop namespace={namespace} />}
```

Comment:
this doesn't look right, if `namespace` is not there then you are passing as a prop to `HeaderTop`, and `HeaderTop` is rendered only if the namespace is not there based on current logic.

Reply:
I totally overlooked this, thanks for catching it